rust/src/dcerpc/dcerpc.rs

```diff
@@ -543,19 +543,6 @@ impl DCERPCState {
         None
     }
 
-    pub fn handle_bind_cache(&mut self, call_id: u32, is_response: bool) {
-        if self.clear_bind_cache == true {
-            self.bind = None;
-            self.bindack = None;
-        }
-        if self.prev_tx_call_id == call_id && is_response == true {
-            self.clear_bind_cache = true;
```

Comment:
Perhaps, we can get rid of this field from DCERPCState struct as it does not seem to be used elsewhere.